protocol-specs/generator_outputs/unlock_transaction/unlock_transaction.json

```diff
@@ -0,0 +1,235 @@
+{
+	"title": "Valid unlock transaction",
+	"summary": "Valid unlock transaction",
+	"config": {
+		"network": "devnet"
+	},
+	"runner": "unlock_transaction",
+	"handler": "unlock_transaction",
+	"testCases": {
+		"input": {
+			"account": {
+				"passphrase": "lava toe nuclear candy erode present guilt develop include type pluck current",
+				"publicKey": "8c3d81b1555fbe4692adfa1026ee21c043633b9369924cf2790e2e0fc6b47a66",
+				"address": "841832338348093031L"
+			},
+			"networkIdentifier": "e48feb88db5b5cf5ad71d93cdcd1d879b6d5ed187a36b0002cc34e0ef9883255",
+			"delegates": [
+				{
+					"passphrase": "vivid phrase noble marble puzzle result pony dream loud deliver catch liquid",
+					"publicKey": "5430e775505b3145c124d15dc7c84ca7c751ecb69faf653bfb1e0c91e6e22f8a",
+					"address": "12957061101390022344L"
+				},
+				{
+					"passphrase": "lonely good salon icon easy awkward cart tape vanish flee cattle spin",
+					"publicKey": "73a56ce40aa991293250d9bd61471d19111f023cf1827c9be189deed733f9ea2",
+					"address": "356975984361330918L"
+				},
+				{
+					"passphrase": "wall stuff hand climb know earn mix type tragic doctor abandon bamboo",
+					"publicKey": "88d1d4e94f2466fe69770a510dc8e6c638875b71e96c02b4791ccc032a2a6472",
+					"address": "7539210577161571444L"
+				},
+				{
+					"passphrase": "since feel friend season leaf thunder garage learn clump negative zone actress",
+					"publicKey": "41583c71f266a84200f0bfdee9b3bb984f6d67f3c903ba7288c97f1259bf8ddc",
+					"address": "15094767118732616261L"
+				},
+				{
+					"passphrase": "pilot payment morning average bread crucial voice donor exchange egg until elite",
+					"publicKey": "5158379dc110c7fc011cfaf52466016668aecc65e5bfa79c2958e16da30490bd",
+					"address": "5912821973123214356L"
+				},
+				{
+					"passphrase": "tuna tide child strategy message snap purpose vibrant erode deputy damage shed",
+					"publicKey": "7cae1f08e4a4a437cffec509951ed1f30451415fff725adaa46a6d8946e95787",
+					"address": "18070133408355683425L"
+				},
+				{
+					"passphrase": "pet later deliver cave weekend shell nerve basket barely tip awful fine",
+					"publicKey": "f7b9ea443bdc180cd4116e2a86e302639b4b41659d818d5011bfff0642453c3a",
+					"address": "3640717344948993040L"
+				},
+				{
+					"passphrase": "episode topic dance ice garbage admit myself wage slim echo owner rifle",
```

Comment:
Why we have to store the passphrase for every delegate? I think we just need to know if unlocking for a particular delegate vote is associated with a valid delegate. For that only delegate list of public keys is enough.

Reply:
I think passphrase is not really needed here. will remove